asgi_ipc.py

```diff
@@ -42,15 +42,11 @@ def __init__(self, prefix="asgi", expiry=60, group_expiry=86400,
         )
         self.thread_lock = threading.Lock()
         self.prefix = prefix
-        self.channel_store = MemoryDict(
-            "/%s-chan" % self.prefix,
-            size=channel_memory
-        )
+        path = Path('/tmp') / Path('{prefix}.sqlite'.format(prefix=prefix))
```

Comment:
Rather than hardcoding to use `/tmp`, which is not always valid, I'd suggest using `tempfile`. Might also remove the need to use `pathlib`